src/graphql.rs

```diff
@@ -1129,6 +1130,10 @@ impl FilterTypeType {
         match &self.entity {
             FilterableType::Scalar(s) => s.name().expect("scalar name should exist"),
             FilterableType::Enum(e) => e.name().expect("enum type name should exist"),
+            FilterableType::List(l) => match l.of_type().unwrap().name() {
```

Comment:
Here, we're grabbing the name of the List's inner type so that we can construct the qualified entity name.

Reply:
Could you please replace the `unwrap` call with an `expect(<error>)`, because any failures due to these become easier to track with unique error messages.

Reply:
Just updated the code as part of https://github.com/supabase/pg_graphql/pull/520/commits/c1a4ebe7f0e6973a10ad66d6b7e1c7a7b2005772, but I'm a Rust newbie, so please do let me know if there's a better way to write this line!

Reply:
@mlabisi the new code looks fine.